### ISSUE TITLE
suggestion: Add Open-Source License

### ISSUE THREAD

**Issue by YieldingExploiter** [exxtremestuffs/SimpleSpySource#15]
Currently, the project has no license file. I'd suggest relicensing it under something like MIT or GPL.

**Comment by YieldingExploiter** [comment]
Without an Open-Source License, redistributing, modifying, or commercially using this software is illegal, without permission from @exxtremestuffs on a case-by-case basis.

**Comment by paygammy** [comment]
I don't think the repository's OP does not have basic knowledge on licensing open-source software. It's been several months and it seems like a push and hide repository to just publish something and leave.

Assuming the developer does not care, which has been proven over the countless months; you shouldn't have to worry about modifying it.

The code is using Roblox's API which in turn leads legal discrepancies towards OP, so I don't see how it's necessarily illegal if Roblox Corporation can file a right to erasure.

**Comment by YieldingExploiter** [comment]
> The code is using Roblox's API which in turn leads legal discrepancies towards OP, so I don't see how it's necessarily illegal if Roblox Corporation can file a right to erasure.

My point was, excluding to the extent permitted by fair use/fair dealings law, the author's work is copyrighted and therefor cannot be forked, or otherwise improved upon, without explicit consent from the author. This removes the freedom for the user, for, in this case, no real reason.

Roblox API or not, this would be the case - If I made something to render a cat on the screen using the Roblox APIs, provided I wrote that code, if I put it on github, without a license, it would be, by default, under the All Rights Reserved license, meaning no-one else can use, modify, redistribute or resell that code without my explicit permission

Licensing said theoretical cat drawing script, under, as an example, the MIT License, would mean anyone can do anything with it, provided they credit me for the original.
